### PR TITLE
Updates and blog post for Discord

### PR DESCRIPTION
**Pull request by aomarks**
- Updates all references and links to Slack with Discord
- Adds a blog post about launching the new Discord server
- This is intended to merge on Tuesday, September 6.

Fixes https://github.com/lit/lit.dev/issues/923
Fixes https://github.com/lit/lit.dev/issues/924

### CODE REVIEW

File: packages/lit-dev-content/site/blog/2022-08-06-discord.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+## What's happening to Slack?
+
+The *Lit & Friends* workspace on Slack will be going into read-only mode on
+**Tuesday, September 13, 2022**. After that, you'll be able to read existing
+messages, but won't be able to post new ones. We've already sent a message to
+every channel about these plans.
+
+We decided to retire the Slack instead of maintaining both because we want to
+encourage more communication and less silos. Slack has served the Lit community
+well, but we think Discord
```
Comment (kevinpschaaf) [inline packages/lit-dev-content/site/blog/2022-08-06-discord.md:52]: Dangling sentence

File: packages/lit-dev-content/site/home/1-splash.html [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <section id="new-banner">
-  <p>📣 Check out our brand-new <a href="{{site.baseurl}}/tutorials">Tutorials</a> page and <a href="{{site.baseurl}}/youtube">Lit YouTube Channel!</a></p>
+  <p>📣 Check out our brand-new <a href="{{site.baseurl}}/discord/">Discord server!</a></p>
```
Comment (kevinpschaaf) [inline packages/lit-dev-content/site/home/1-splash.html:2]: I've felt that as rendered, the space between the megaphone and the first word is a few px too crowded... anyone else?

**Review by e111077 — APPROVED** [review]
stuff other than the content of the blogpost LGTM

File: packages/lit-dev-content/site/blog/2022-08-06-discord.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+<!--
+## Can I separate my work and personal life on Discord?
+
+Many of us already Discord for our own personal communities, like gaming. If you
```
[the remark this answers is not in the document]
Comment (aomarks) [reply packages/lit-dev-content/site/blog/2022-08-06-discord.md:57]: Oh, yeah I meant to delete it.

File: packages/lit-dev-content/site/home/1-splash.html (hunk 2 above)
Comment (e111077) [reply packages/lit-dev-content/site/home/1-splash.html:2]: the megaphone on mac isn't very legible anyway due to contrast 🤷‍♂️

Comment (aomarks) [reply packages/lit-dev-content/site/home/1-splash.html]: I made some tweaks to the banner, PTAL!

image


- Used the darker Lit blue for the background. Improves contrast. The text is kinda on the lighter blue is kinda hard for me to read, as well as the emoji not being super legible.

- Increased font size. Decreased padding.

**Pull request merged**